vkconfig/configurator.cpp

```diff
@@ -1560,21 +1561,21 @@ void Configurator::SetActiveConfiguration(Configuration *configuration) {
     // Loop through all the layers
     for (int layer_index = 0; layer_index < configuration->_layers.size(); layer_index++) {
         LayerFile *layer_file = configuration->_layers[layer_index];
-        stream << endl;
-        stream << "# " << layer_file->_name << endl;
```

Comment:
Curious why you replaced the endl with "\n"?

Reply:
On my machin using QtCreator, I have an error on this.

Reply:
Hmmm. Odd, but plenty good enough reason.